### PR TITLE
Show the pretty format name in the battle tab

### PR DESCRIPTION
**Pull request by urkerab**
Tabs for battles currently show the format id and the players. I thought it would look slightly nicer if we used the format name instead. Example:

    _______________________
    |      ounomega      X|
    |junzheng5 vs. OVOOHKO|
    _______________________
    |    OU (no Mega)    X|
    |junzheng5 vs. OVOOHKO|

### CODE REVIEW

File: js/client.js [hunk 1]
```diff
@@ -1893,7 +1893,7 @@
 							name = '(empty room)';
 						}
 					}
-					return buf + '<i class="text">' + formatid + '</i><span>' + name + '</span></a><a class="closebutton" href="' + app.root + id + '"><i class="fa fa-times-circle"></i></a></li>';
+					return buf + '<i class="text">' + format + '</i><span>' + name + '</span></a><a class="closebutton" href="' + app.root + id + '"><i class="fa fa-times-circle"></i></a></li>';
```
Comment (Slayer95) [inline js/client.js]: Escape on render

Comment (urkerab) [inline js/client.js]: So you'd prefer me to inline the `Tools.escapeFormat(id.split('-')[1])` directly?

Comment (Slayer95) [inline js/client.js]: Yup, but I myself would keep ``formatid``'s variable around for readibility.

Comment (Zarel) [inline js/client.js]: I assume Slayer95 means `Tools.escapeFormat(formatid)`. That's what I'd prefer here.

Comment (Zarel) [inline js/client.js]: Apparently `format` is already defined but unused? We can get rid of that.

**Comment by urkerab** [comment]
(sorry for the delay, was working on PR #507)

**Pull request merged**